extensions/shared/src/main/java/app/revanced/extension/youtube/patches/components/ReturnYouTubeDislikeFilterPatch.java

```diff
@@ -145,6 +143,12 @@ public boolean isFiltered(String path, @Nullable String identifier, String allVa
             // Must pass a null id to correctly clear out the current video data.
             // Otherwise if a Short is opened in non-incognito, then incognito is enabled and another Short is opened,
             // the new incognito Short will show the old prior data.
+
+            ShortsInformation.setShortId(matchedVideoId);
```

Comment:
Why don't u rename this filter patch to ShortsVideoIdFilterPatch or sth like that xD

Reply:
I'm waiting inotia to clarify if this approach is valid